adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequest.java

```diff
@@ -31,7 +31,7 @@
 /**
  * Represent request and keeps authorization code and similar info.
  */
-class AuthenticationRequest implements Serializable {
+public class AuthenticationRequest implements Serializable {
```

Comment:
why make AuthenticationRequest and its constructors public ? since it's used by ad-accounts, and there's a class extending it.  consumer is : ad-accounts/AADAuthenticator/src/main/java/com/microsoft/workaccount/workplacejoin/BrokerContext.java:676

Reply:
This must not be public. Leave it in package scope and we can work around it in BrokerContext

Reply:
changed it back.